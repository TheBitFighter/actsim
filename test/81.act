defproc test()
{
  int<4> x;
  chp {
    x := x{0..1}
  }
}
