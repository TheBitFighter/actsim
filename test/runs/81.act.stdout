[                  10] <>  bit-field {0..1} is invalid; using one-bit result
